superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -83,6 +83,31 @@ def add_parser_arguments(self):
             help='Iterations of the benchmark. Default is 3.',
         )
 
+        self._parser.add_argument(
+            '--numa',
+            type=int,
+            default=-1,
```

Comment:
Maybe we don't need `default=-` since `required=False`. When not specified, `args.numa` will be `None`.